CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
-## 4.0.1
+## 5.0.0
+- Added support for new TLD's ( .888, .nft, .coin, .blockchain, .wallet, .x, .bitcoin, .dao )
+- Replace CNS with UNS
+- Rename Eip1993Factories -> Eip1193Factories
+- Introduce ZilliqaProvider for ZNS
+- Add new method `Resolution#registryAddress(domain: string): Promise<string>` 
+- Return Ens support
+- Updated network-config
```

Comment:
@JohnnyJumper What does it mean for end user?

Reply:
Just informing that the default setting has been updated.
I will update the message to something more clear.